Website/Resources/Search/SearchSkinObjectPreview.js

```diff
@@ -96,7 +96,9 @@
 	                    var $searchButton = $wrap.parents('.SearchContainer').length ? $wrap.parent().next() : $wrap.next();
                         var href = $searchButton.attr('href');
                         var code = href.replace('javascript:', '');
-                        eval(code);
+                        setTimeout(function(){
+                            eval(code);
```

Comment:
Hi @zyhfish , this eval code is about extracting a javascript function to execute. We could replace:
-  var href = $searchButton.attr('href');
to
-  $searchButton.[0].click();

It would execute the same function without eval();